### DISCUSSION TITLE
Support for color codes in anvils

### DISCUSSION THREAD

**Discussion by notaduck448** [EssentialsX/Essentials#4259] [Ideas and Feature Suggestions] [answered]
It would be nice to add support for color and formatting codes when renaming items in anvils. Permissions could be used to control access to this feature like the `essentials.signs.color` and `essentials.signs.rgb` permissions.

**Comment by JRoy** [comment, marked as answer]
The /itemname command is here for this purpose.

**Reply by notaduck448** [reply]
That's true, but I don't want to give players access to the command because it lets them rename items wherever they are without an anvil. I still want to keep the game mechanic of using anvils to rename items, but give the option of using color codes in them.
